packages/nextjs/app/builders/[address]/_components/UserProfileCard.tsx

```diff
@@ -61,6 +67,17 @@ export const UserProfileCard = ({ user, batch }: { user: NonNullable<UserByAddre
           <div className="text-sm text-neutral">
             Joined {new Date(user.createdAt).toLocaleDateString("en-US", { month: "long", year: "numeric" })}
           </div>
+
+          <div>
+            <p className="mt-0 mb-1 text-center">
+              {experiencePoints.userPoints} / {experiencePoints.totalPoints} XP points
```

Comment:
I think `XP points` is redundant? And should just be XP

Reply:
Going to try just "Points", since XP is too small and gets lost